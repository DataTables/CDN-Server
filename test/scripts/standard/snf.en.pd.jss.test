snf/en/pd-4.0.0/test.js
200
