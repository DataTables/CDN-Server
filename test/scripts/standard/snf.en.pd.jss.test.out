extensionsURL >snf/en/pd-4.0.0<
extensionsList >localeEn en, ProductD 4.0.0<
source >"snf/en/pd-4.0.0"<
testString >test string<
./ProductD-4.0.0/productd.styleNoFiles.en.js


